stack.py

```diff
@@ -46,15 +48,15 @@ def __eq__(self, other):
     '>': 'gt',
     '<=': 'le', '≤': 'le',
     '>=': 'ge', '≥': 'ge',
-    '∅': 'nop',
+    '∅': 'nop'
```

Comment:
You don't necessarily need to change this back, but just a note that I like to leave trailing commas so that adding new items doesn't cause a diff on an irrelevant line.

e.g., if you have
```
foo,
bar
```
and you change it to
```
foo,
bar,
baz
```
the diff looks like
```diff
 foo,
-bar
+bar,
+baz
```
when with a trailing comma already there it could just look like
```diff
 foo,
 bar,
+baz,
```